examples/postgresql-ha/main.tf

```diff
@@ -81,6 +81,11 @@ module "pg" {
     start_time                     = "20:55"
     location                       = null
     point_in_time_recovery_enabled = false
+    transaction_log_retention_days = null
+    backup_retention_settings = {
```

Comment:
I don't think we need to nest this. Let's just collapse to the `backup_configuration` level:

```
backup_configuration = {
    enabled                        = true
    start_time                     = "20:55"
    location                       = null
    point_in_time_recovery_enabled = false
    transaction_log_retention_days = null
    retained_backups = 365
    retention_unit = "COUNT"
}

Reply:
@morgante Good idea, updated in 8676fba 👍